pkg/meta/model/job_args.go

```diff
@@ -482,6 +482,50 @@ func GetResourceGroupArgs(job *Job) (*ResourceGroupArgs, error) {
 	return getOrDecodeArgsV2[*ResourceGroupArgs](job)
 }
 
+// DropColumnArgs is the arguments of dropping column job.
+type DropColumnArgs struct {
+	ColName  pmodel.CIStr `json:"column_name,omitempty"`
+	IfExists bool         `json:"if_exists,omitempty"`
+	// indexIDs is used to make sure we don't truncate args when decoding the rawArgs.
```

Comment:
```suggestion
```

Seems you copied this comment from other place, but it's just a workaround for version 1 behaviours. We don't need it now.